accounts-db/src/accounts_db/geyser_plugin_utils.rs

```diff
@@ -124,6 +124,7 @@ impl AccountsDb {
             // As long as all accounts for this slot are in 1 append vec that can be iterated oldest to newest.
             self.notify_filtered_accounts(
                 slot,
+                i as u64,
```

Comment:
Is `i` the same as write_version? 
It seems that `i` is the index of the accounts in the append-vec. It is not the times that *one* particular account were written in the appendvec.

Reply:
is `write_version` a per account concept? different accounts can have the same write version, right?

Reply:
Maybe it is a question for @lijunwangs ?

Reply:
> // write_version is only used to order multiple entries with the same pubkey,

Never mind, from the comments above, it seems that we don't promise that write_version will be sequential. Therefore, passing the counter would be fine.